src/services/gameSettingsMessage.ts

```
@@ -106,7 +140,7 @@ export default async function sendGameSettingsMessage({
   tiles,
   customTiles = [],
   reason = '',
-}) {
+}: SendMessageOptions): Promise<DocumentReference<DocumentData> | void> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)